app/models/id_card_attributes.rb

```diff
@@ -16,12 +16,12 @@ def traits
       'edipi' => @user.edipi,
```

Comment:
do we know why these are using strings v. symbols?

Reply:
Dunno, looks to me like it's because these are real close to the controller, so it's basically a serializer at this point. It looks like `id_card_attributes_controller` asks for these, barely massages them, and then renders it.

We use strings in SO many places where we could use symbols. Worth an investigation in our general code base for sure